public/views/signin.html

```diff
@@ -1,71 +1,91 @@
+                  <div class="form-group">
+                    <label class="text-right col-md-3" for="pwd">Password:</label>
+                    <input class="col-md-5 input-text" type="password" class="form-control"
+                      ng-model="password" required novalidate>
+                  </div>
+                </div>
+                <div class="text-center" style="padding-top: 1em">
+                  <button type="submit" class="btn btn-dark" style="width:8em; cursor:pointer">Sign In</button>
+                </div>
+              </form>
 
-      <div class="sign-body">
-        <div class="more-sign">
-          <div class="sign-error" ng-show="showError() === 'invalid'">
-            Please check your username/password and try again.
-          </div>
-          <form class="signup form-horizontal" ng-submit="signIn()" method="post">
-            <div class="control-group">
-              <label for="email" class="control-label"></label>
-              <div class="controls">
-                <input class="input-info" id="email" type="text" ng-model="email" name="email" placeholder="Email" required novalidate />
+              <div class="row" style="padding-top: 3em">
+                <hr class="col-md-4 text-right">
+                <p class="col-md-1"> OR </p>
+                <hr class="col-md-4 text-left">
               </div>
-            </div>
-            <div class="control-group">
-              <label for="password" class="control-label"></label>
-              <div class="controls">
-                <input class="input-info" id="password" type="password" ng-model="password" name="password" placeholder="Password" required novalidate />
+
+              <div class="text-center" style="padding-top: 1em">
+                <p> Sign in with:
+                  <a href="/auth/facebook">
+                    <i class="fa fa-facebook icons fa-2x" aria-hidden="true"></i>
+                  </a>
+                  <a href="/auth/github">
+                    <i class="fa fa-github icons fa-2x" style="padding-left: 5px" aria-hidden="true"> </i>
+                  </a>
+                  <a href="/auth/google">
+                    <i class="fa fa-google icons fa-2x" style="padding-left: 5px" aria-hidden="true"></i>
+                  </a>
+                  <a href="/auth/twitter">
+                    <i class="fa fa-twitter icons fa-2x" style="padding-left: 5px" aria-hidden="true"></i>
+                  </a>
+                </p>
               </div>
+
+              <p class="text-center" style="font-size:13px; padding-top: 1em"> Play as guest?
+                <a href="#"> New Game </a>
+              </p>
             </div>
-            <div class="form-actions top-margin">
-              <button id="sign-up-btn-other" type='submit'>Sign In</button>
-            </div>
-          </form>
+          </div>
         </div>
       </div>
     </div>
-    </div>
-    </div>
   </div>
-</div>
-
+</div>
```

Comment:
Add new line to end of file. It will be much easier to set your editor to add it automatically when saving. If using Visual Studio Code, then add this to your settings:
```json
{
   "files.trimTrailingWhitespace": true,
   "files.insertFinalNewline": true,
   "files.trimFinalNewlines": true 
 }